graylog2-web-interface/src/views/components/dashboard/DashboardsOverview/DashboardActions.test.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+
+import { render, waitFor, screen } from 'wrappedTestingLibrary';
+import * as React from 'react';
+import userEvent from '@testing-library/user-event';
+import { PluginStore } from 'graylog-web-plugin/plugin';
+
+import { asMock } from 'helpers/mocking';
+import { ViewManagementActions } from 'views/stores/ViewManagementStore';
+import DashboardActions from 'views/components/dashboard/DashboardsOverview/DashboardActions';
+import { simpleView } from 'views/test/ViewFixtures';
+
+jest.mock('views/stores/ViewManagementStore', () => ({
+  ViewManagementActions: {
+    delete: jest.fn(),
+  },
+}));
+
+describe('DashboardActions', () => {
+  let oldWindowConfirm;
+
+  const simpleDashboard = simpleView();
+
+  const clickDashboardAction = async (dashboardId: string, action: string) => {
+    const actionsButton = (await screen.findAllByTestId(`dashboard-actions-dropdown-${dashboardId}`))[0];
```

Comment:
Is that possible that we might have several items with the same `data-testid `?  Maybe it would be enough `screen.findByTestId`?

Reply:
Thanks for pointing it out, these were `DashboardPage` tests before, now where we have a separate file for `DashboardActions` test we can get rid of the `data-testid`.